src/memory-analyzer/gdb_api.cpp

```diff
@@ -56,6 +57,39 @@ gdb_apit::~gdb_apit()
   wait(NULL);
 }
 
+size_t gdb_apit::query_malloc_size(const std::string &pointer_expr)
+{
+  write_to_gdb("-var-create tmp * malloc_usable_size(" + pointer_expr + ")");
```

Comment:
I believe `malloc_usable_size` is only available on Linux. On OSX I think the equivalent function on Mac OS X is `malloc_size` - so we probably need to do some conditional compilation guarding here...

Reply:
I thought the machine interface worked so much differently under OSX that we only wanted to compile there but not actually run the `memory-analyzer`. Anyway what do you think about https://github.com/diffblue/cbmc/pull/4482/commits/57f08e14344316b19c9ae5ecc828d241f28d27be?